app/_src/gateway/production/deployment-topologies/hybrid-mode/incremental-config-sync.md

```diff
@@ -0,0 +1,134 @@
+
+## Enable incremental config sync
+
+You can enable incremental config sync when installing {{site.base_gateway}} in [hybrid mode](/gateway/{{page.release}}/production/deployment-topologies/hybrid-mode/setup/), or when setting up a {{site.konnect_short_name}} Data Plane.
+
+{:.important}
+> **Caution**: There are some [limitations for custom plugins](#using-incremental-config-sync-with-custom-plugins) and for 
+[rate limiting plugins](#using-incremental-config-sync-with-rate-limiting-plugins) when using incremental config sync. 
+Review and adjust your plugin config before enabling this feature.
+
+During setup, set the following values in your `kong.conf` files on both Control Planes and Data Planes:
+
+```
+cluster_rpc = on
+cluster_rpc_sync = on
+```
+
+Or, if running {{site.base_gateway}} in Docker, set the following environment variables:
+```
+export KONG_CLUSTER_RPC_SYNC=on
+export KONG_CLUSTER_RPC=on
+```
+
+## Limitations
+
+### Using incremental config sync with custom plugins
+
+When incremental config sync is enabled, the configuration change notification from the Control Plane only triggers an event for changed entities, and doesn't trigger cache updates in Data Plane nodes. 
+This causes outdated and inconsistent configuration for custom plugins.
+
+If you are running {{site.base_gateway}} on {{site.konnect_short_name}} or in hybrid mode, you need to adjust your custom plugins to be compatible with incremental config sync.
+
+#### Workaround for custom plugins
+
+To ensure your custom plugin configuration is kept up to date, you must add additional code logic to register the CRUD events for the entities the plugin cares about, and invalidate the relevant cache data.
+
+Add the code at the end of the `init_worker` function for the custom plugin.
+For example:
+
+
+```lua
+function _M.init_worker()
+  -- ...
+  -- ... your original custom code
+  -- add the example code below AFTER any custom plugin code
+ 
+  -- Check if worker_events can be registed successfully
+  if not (kong.worker_events and kong.worker_events.register) then
+    return
+  end
+
+  -- Check the deployment mode and incremental sync feature is enabled
+  if kong.configuration.database == "off" and not kong.sync then
+    return
+  end
+  
+  -- Do the event registration and force invalidate the corresponding cache
+  kong.worker_events.register(
+    function(data)
+    -- logic to identify what cache need to be invalidated
+    kong.cache:invalidate(CACHE_KEY) -- Your plugin logic to invalidate the cache
+  end, "crud", "ENTITY_NAME") -- Register the events for entity the plugin cares
+  
+  -- Repeat the register events for other ENTITIES
+  
+end
+```
+
+### Using incremental config sync with rate limiting plugins
+
+We don't recommend using the `local` strategy for rate limiting plugins with incremental config sync.
```

Comment:
```suggestion
We don't recommend using the `local` strategy for [rate limiting plugins](/hub/?search=rate%2520limiting) with incremental config sync.
```
I'm not sure if that link will work. Ideally, this would be the dev site RL landing page :)